components/landingNavbar/index.js

```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { useState } from "react";
+
+const NavbarContainer = styled.nav`
+  position: fixed;
+  top: 200px;
+  left: 10px;
+  width: 150px;
+  height: 300px;
+  background-color: #f2f2f2;
+  padding: 1rem;
+  transition: left 0.3s ease-in-out;
+  border: black solid 1px;
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
+`;
+const NavbarLinks = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+const NavbarLink = styled(Link)`
+  color: #333;
+  text-decoration: none;
+  margin-bottom: 0.5rem;
+  border: solid black 1px;
+  border-radius: 100px;
+  padding: 10px;
+  background-color: #ff9900;
+
+  &:hover {
```

Comment:
do we use hover when designing for mobile apps? I think you'll be fine leaving it in, but you don't need it here. A finger touching a screen can not "hover" 🙃

Reply:
True, but i think it can stay for the pc users ;)